container/lxd/manager.go

```diff
@@ -88,20 +84,20 @@ func NewContainerManager(cfg container.ManagerConfig, cSvr lxd.ContainerServer)
 }
 
 // Namespace implements container.Manager.
-func (manager *containerManager) Namespace() instance.Namespace {
```

Comment:
Unfortunately this introduces a lot of mechanical changes along with the actual content changes. But I'll deal. It did make me think I was looking at the wrong patch when you said "ignore the mechanical changes" and then this one starts off with 30 lines of 's/manager/m/' && s/instance/container/

Reply:
Yes; that should have been isolated.